GVRf/Framework/src/org/gearvrf/GVRSceneObject.java

```diff
@@ -558,6 +558,18 @@ public void removeChildObject(GVRSceneObject child) {
     }
 
     /**
+     * Check if {@code otherObject} is colliding with this object.
+     * 
+     * @param otherObject
+     *            {@link GVRSceneObject Object} to check for collision with this
+     *            object.
+     * @return True if objects collide and False otherwise.
```

Comment:
In Java, it's `true` and `false`, or `{@code true}` and `{@code false}` if you don't want your JavaDoc to stick out like sore thumb.